### PR TITLE
New Arabic source sunovels.com

### PR DESCRIPTION
**Pull request by Dredd1987**
Adding the new source, sunovels.com, has resolved issue #130.

However, there's a problem with images on the library and search pages. It takes a while for the requested URLs to show up in the HTML. There might be a more efficient way to get these images, maybe through an API or some other method. I haven’t found a good solution.

### CODE REVIEW

**Comment by K1ngfish3r** [comment]
image
get this script tag and do the json magic again?

**Comment by rajarsheechatterjee** [comment]
@Dredd1987 Thank you for contributing to the Arabic plugins. ✨

**Comment by Dredd1987** [comment]
I couldn't figure out the HTTP endpoint for the JSON, and the JSON within the script tag is a hassle to clean up, so I used regex and matching. It's not elegant, but it works.

File: src/plugins/arabic/sunovels.ts [hunk 1]
```diff
@@ -0,0 +1,306 @@
+import { CheerioAPI, load as parseHTML } from 'cheerio';
+import { fetchApi } from '@libs/fetch';
+import { Plugin } from '@typings/plugin';
+import { Filters, FilterTypes } from '@libs/filterInputs';
+import { defaultCover } from '@libs/defaultCover';
+
+class RewayatClub implements Plugin.PagePlugin {
```
Comment (K1ngfish3r) [inline src/plugins/arabic/sunovels.ts:7]:
```suggestion
class Sunovels implements Plugin.PagePlugin {
```
Wrong plugin :)

File: src/plugins/arabic/sunovels.ts [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+        { label: 'مكتمل', value: 'Completed' },
+        { label: 'جديد', value: 'New' },
+        { label: 'مستمر', value: 'Ongoing' },
+      ],
+      type: FilterTypes.Picker,
+    },
+  } satisfies Filters;
+}
+
+export default new RewayatClub();
```
Comment (K1ngfish3r) [inline src/plugins/arabic/sunovels.ts:299]:
```suggestion
export default new Sunovels();
```
yes

**Review by rajarsheechatterjee — APPROVED** [review]

**Pull request merged**